### PR TITLE
PR for navigation component!

### PR DESCRIPTION
**Pull request by adidoshi**
### Hey! I created the simple desktop navigation component, please review it.

### CODE REVIEW

File: components/navigation/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+.nav-center .input {
+  padding: 0.6rem;
+}
+
+.navigation .nav-right {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 0.5em 0;
```
Comment (kiranpratyush) [inline components/navigation/Navigation.css:24]: We can omit 0 here

**Review by kiranpratyush — APPROVED** [review]
Good work👍👍

File: components/navigation/Navigation.css (hunk 1 above)
Comment (adidoshi) [reply components/navigation/Navigation.css:24]: Thanks will make the changes!

File: components/navigation/Navigation.html [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+          </p>
+          <hr class="example-hr" />
+          <h2>Simple Navigation</h2>
+          <p class="example-desc">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Adipisci,
+            nobis.
+          </p>
+
+          <div class="utilities-box">
+            <div class="navigation">
```
Comment (msdeshmukh009) [inline components/navigation/Navigation.html:175]: You can directly use `<nav>` tag here if you want.

**Review by msdeshmukh009 — APPROVED** [review]

**Review by vijaytembugade — APPROVED** [review]
Looks good adi!

File: components/navigation/Navigation.html (hunk 2 above)
Comment (adidoshi) [reply components/navigation/Navigation.html:175]: Yes, thought of that, will make the change.

**Pull request merged**